# Goal
string motion_name
float32 time_factor
---
# Result
int32[] error_code
int32 SUCCESSFUL = 0
int32 INVALID_GOAL = -1
int32 INVALID_JOINTS = -2
int32 OLD_HEADER_TIMESTAMP = -3
int32 PATH_TOLERANCE_VIOLATED = -4
int32 GOAL_TOLERANCE_VIOLATED = -5
int32 MOTION_UNKNOWN = -6

string error_string
string[] error_group
---
# Feedback

